bluepysnap/circuit_ids.py

```diff
@@ -253,6 +253,26 @@ def unique(self, inplace=False):
         """
         return self._apply(lambda x: x.unique(), inplace)
 
+    def intersection(self, ids, inplace=False):
+        """Take the intersection of this CircuitIDs with another.
+
+        Notes:
+            This function returns a CircuitIDs which contains only those ids which are in both
+            this CircuitIds and the provided CircuitIds. The index of the result will be sorted.
```

Comment:
Is there a specific reason why this is sorted by default?

Reply:
would it be better if it did not sort?

Reply:
I don't know, really. In the very worst scenario, sorting could add quite the overhead but realistically thinking, in most cases the ids are likely sorted or mostly sorted anyway. Let's keep it sorted for now.